tests/DialogServiceConnectorTests.ts

```diff
@@ -175,6 +180,8 @@ test("Output format, default", () => {
     expect(dialogConfig.outputFormat === sdk.OutputFormat.Simple);
 });
 
+// Skip this test pending parity check with C++/C# -- 1006/404 is only returned when an unintended double-forwardslash
+// is present in the connection URL.
 test("Create BotFrameworkConfig, invalid optional botId", (done: jest.DoneCallback) => {
```

Comment:
test.skip seems appropriate here

Reply:
Incomplete removal! I reenabled this test after correcting the query string parameter keys (after which it passes again), but forgot to remove the comment. Good eye and thanks for pointing this out!